### PR TITLE
Test/dx contract calls

### PR DESCRIPTION
**Pull request by Velenir**
This includes tests for

> DX.updateThresholdNewTokenPair(uint _thresholdNewTokenPair)
> DX.updateThresholdNewAuction(uint _thresholdNewAuction)

> ------------------------

>  DX.startMasterCopyCountdown(address _masterCopy)
> // //timeskip 30 days
> DX.updateMasterCopy()
> 
>-------------------------------

> DX.updateApprovalOfToken(address[] token, bool approved)
> 
>----------------
> DX.initiateEthUsdOracleUpdate(PriceOracleInterface _ethUSDOracle)
> // //timeskip 30 days
> DX.updateEthUSDOracle()
> 
>--------------
> DX.updateAuctioneer(address _auctioneer)

Which currently don't work because when `GenesisProtocol._execute` calls `            ProposalExecuteInterface(proposal.callbacks).executeProposal`, it's as if no code is executed
That is evident in log events: `ExecuteProposal, GPExecuteProposal` appear in the logs. but `ProposalDeleted and ProposalExecuted` don't.

Test by running `npx truffle test test/call_DX_methods.js`

### CODE REVIEW

File: test/call_DX_methods.js [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+//     const txResult = await dxReputation.mint(account, amount)
+//     console.log(`REP minted:
+// - Transaction: ${txResult.tx}
+// - Gas used: ${txResult.receipt.gasUsed}
+// `)
+//   }
+// }
+const callForAll = (accounts, cb) => { return Promise.all(accounts.map(cb)) }
+
+contract('Execute updateAuctioneer proposal', (accounts) => {
```
Comment (orenyodfat) [inline test/call_DX_methods.js]: where does the GenericS parameters set is perform ?

Comment (orenyodfat) [reply test/call_DX_methods.js]: can you please add a call to validate the contractToCall parameter ?

Comment (Velenir) [reply test/call_DX_methods.js]: Some parameters were indeed incorrectly set. And auctioneer was different from the expected one.
Thank you for pointing me in this direction.

Tests now work, proposals execute and final values are what is expected, but there's still no `ProposalDeleted and ProposalExecuted` events. Do you have any idea why?

Comment (Velenir) [reply test/call_DX_methods.js]: Parameters are set in migration 13

**Pull request merged**